TezosKit/Client/TezosNodeClient.swift

```diff
@@ -603,12 +607,50 @@ public class TezosNodeClient: AbstractClient {
     }
   }
 
-  /**
-   * Retrieve metadata needed to forge / pre-apply / sign / inject an operation.
-   *
-   * This method parallelizes fetches to get chain and address data and returns all required data
-   * together as an OperationData object.
-   */
+  /// Parse a preapplication RPC response and extract an error if one occurred.
+  internal static func preapplicationError(from preapplicationResponse: [[String: Any]]) -> TezosKitError? {
+    let contents: [[String: Any]] = preapplicationResponse.compactMap { operation in
+     operation["contents"] as? [[String: Any]]
```

Comment:
Update "contents", "metadata", etc... to be constants declared, and commented, somewhere?